src/main/resources/db/changelog/changes/v1.1.0/insert-instances-definition.xml

```diff
@@ -0,0 +1,120 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="MODFQMMGR-129" author="mweaver@ebsco.com" runOnChange="true">
```

Comment:
We already know there will be more fields to add, so I went ahead and preemptively added the `runOnChange`